charts/lily/templates/statefulset.yaml

```diff
@@ -40,7 +40,7 @@ spec:
       - name: regcred
       initContainers:
       - name: initfs
-        image: {{ required "(root).image.repo expected" .Values.image.repo }}:{{ required "expected image tag to be defined" .Values.image.tag }}
+        image: {{ required "(root).image.repo expected" .Values.image.repo }}:{{ default .Chart.AppVersion .Values.image.tag }}
```

Comment:
Think this default needs to be prefixed with `v`

Reply:
Indeed. Good catch. 🙇